BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -797,7 +797,7 @@ def main():
     group.add_argument('-cl', dest='comp_list',nargs='+', help='List of each component files, following XXXX:FileName format')
     cmd_display.add_argument('-t', dest='img_type',  type=str, default='CLASSIC', help='Container Image Type : [NORMAL, CLASSIC, MULTIBOOT]')
     cmd_display.add_argument('-o', dest='out_path',  type=str, default='.', help='Container output directory/file')
-    cmd_display.add_argument('-k', dest='key_path',  type=str, default='', help='Input key directory/file')
+    cmd_display.add_argument('-k', dest='key_path',  type=str, default='', help='Input key directory/file. Use directoy for Container layout as intput file ')

```

Comment:
If there is no layout file, then you expect a 'key_file_name' to be provided here. Isn't it?
In that case, this help text is mis-leading.

Also, too many syntax errors.

"Input key directory/file. Use directoy for Container layout as intput file "

Reply:
Updated decsriptions.